plugin/helpers_requirements.go

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package plugin
+
+import (
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mattermost-server/utils"
+)
+
+func (p *HelpersImpl) CheckRequiredConfig(requiredConfig, actualConfig *model.Config) (bool, error) {
+	return utils.CheckRequiredConfig(requiredConfig, actualConfig)
```

Comment:
Instead of making the plugin pass the current config, could we simplify to fetch it here ourselves via `p.API.GetConfig()`?